gooddata-java-model/src/main/java/com/gooddata/sdk/model/settings/SettingItem.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright (C) 2007-2021, GoodData(R) Corporation. All rights reserved.
+ */
+package com.gooddata.sdk.model.settings;
+
+import com.fasterxml.jackson.annotation.*;
+import com.gooddata.sdk.common.util.GoodDataToStringBuilder;
+
+import static com.gooddata.sdk.common.util.Validate.notEmpty;
+import static com.gooddata.sdk.common.util.Validate.notNullState;
+
+@JsonTypeName("settingItem")
+@JsonTypeInfo(include = JsonTypeInfo.As.WRAPPER_OBJECT, use = JsonTypeInfo.Id.NAME)
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class SettingItem {
+
+    private final String name;
```

Comment:
I would rename it to `key` since it's the correct naming for these settings and provide both `getKey` and `getName` for convenience.

Reply:
Renamed